pkg/skaffold/initializer/builders.go

```diff
@@ -128,7 +128,7 @@ func detectBuilders(enableJibInit, enableBuildpackInit bool, path string) ([]Ini
 		// Check for buildpacks
 		if buildpacks.Validate(path) {
 			results := []InitBuilder{buildpacks.ArtifactConfig{File: path}}
-			return results, true
+			return results, false
```

Comment:
I'm not sure I agree with this. E.g. I could imagine skaffold init to find all the go.mod files under the skaffold repo that are in the examples folders. But maybe I'm wrong and the user should then go into all the individual folders and run the detection? 
I thought we could have some go specific logic that filters on `vendor` being in the path. WDYT?

Reply:
I had the same hesitation. I went to the easy solution and assumed the user could indeed configure things manually in case of a more complicated project layout.

Reply:
However for the IDE use case, detecting everything in the workspace is useful